apps/verify/translations/src/en/fields.json

```diff
@@ -3,12 +3,12 @@
     "legend": "You can use a biometric residence permit (BRP) number or a unique application number (UAN) to sign in to this form.",
     "options": {
       "brp": {
-        "label": "Using a BRP number",
-        "hint": "For example, RZ1234567"
+        "label": "Using a biometric residence permit (BRP) number",
+        "hint": "For example, RZ123456"
```

Comment:
I haven't mentioned this to Jake and Luke yet but this BRP example is actually missing a digit and would be invalid - see discussion in the ticket [ACRS-32](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-32)

Reply:
and i figured out that the first 2 letter are different from the image as well